packages/mobile/src/screens/sign-on-screen/components/AccountHeader.tsx

```diff
@@ -65,7 +65,8 @@ export const AccountHeader = (props: AccountHeaderProps) => {
         style={css({
           position: 'absolute',
           left: spacing.unit4,
-          top: spacing.unit10
+          top: spacing.unit10,
+          width: Dimensions.get('window').width - 4 * spacing.unit4
```

Comment:
`4 * unit4` vs `unit16`
or maybe a comment to explain why?

Reply:
oh yeah good point! ill add a comment and update to unit16